qemu/tests/timerdevice_boot.py

```diff
@@ -169,6 +182,15 @@ def update_clksrc(session, clksrc):
     if params["os_type"] == "linux":
         verify_timedrift(session, is_hardware=True)
 
+    repeat_nums = int(params["repeat_nums"])
+    if repeat_nums:
+        sleep_time = params["sleep_time"]
+        for index in range(repeat_nums):
+            time.sleep(int(sleep_time))
+            verify_timedrift(session)
+            if params["os_type"] == "linux":
+                verify_timedrift(session, is_hardware=True)
```

Comment:
Do not remember this clearly, only check hardware time with linux now ?

Reply:
No, line190 is for checking system time.

Reply:
some misunderstanding, my point is we only verify hardware time with linux guest, no means linux guest only check hardware time.
Anyway, we have reached an agreement,thanks!